recipes/spaghetti-bolognese.html

```diff
@@ -8,7 +8,7 @@
 </head>
 <body>
     <h1>Spaghetti Bolognese</h1>
-    <img src="images/spaghettibolognese.jpeg">
+    <img height="300" src="images/Spagetti-Putaneska.jpeg">
```

Comment:
file names should be kebab-case(e.g. spagetti-putaneska)

Reply:
addressed